backend/src/schema/resolvers/users.js

```diff
@@ -22,6 +22,7 @@ export const getBlockedUsers = async context => {
 }
 
 export const getBlockedByUsers = async context => {
+  if (context.user.role === 'moderator' || context.user.role === 'admin') return []
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-02T13:50:36Z" title="Wednesday, October 2nd 2019, 3:50:36 pm +02:00">Oct 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -22,6 +22,7 @@ export const getBlockedUsers = async context => {
 }
 
 export const getBlockedByUsers = async context => {
+  if (context.user.role === 'moderator' || context.user.role === 'admin') return []
```


We should probably export that check in `permissionsMiddleware` in the future